api/Attr.json

```diff
@@ -103,6 +103,53 @@
           }
         }
       },
+      "name": {
```

Comment:
Surprising that this was missing, but it's listed in https://developer.mozilla.org/en-US/docs/Web/API/Attr with red link color, and I've confirmed it's *not* incorrectly in Node.json already, so this looks good!